open_fdd/version.py

```
@@ -0,0 +1,113 @@
+    m = _CARGO_VERSION_RE.search(text)
+    return m.group(1) if m else None
+
+
+def _catalog_hashes() -> tuple[str | None, str | None]:
+    try:
+        from open_fdd.catalog import effective_config_hash, rule_catalog_hash
+    except ImportError:
+        return None, None
+    try:
+        return rule_catalog_hash(), effective_config_hash()
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)